lib/fizzy/parser.cpp

```diff
@@ -182,6 +182,24 @@ struct parser<Export>
     }
 };
 
+template <>
+struct parser<Data>
+{
+    parser_result<Data> operator()(const uint8_t* pos)
+    {
+        MemIdx memidx;
```

Comment:
```suggestion
        MemIdx mem_idx;
```

Reply:
The type is called MemIdx, why not mirror that in the var name?

Reply:
I don't insist, but in other places (in execute.cpp) we use underscore with `_idx`